ui/widgets/autocomplete.js

```diff
@@ -614,6 +614,22 @@ $.widget( "ui.autocomplete", {
 			// Prevents moving cursor to beginning/end of the text field in some browsers
 			event.preventDefault();
 		}
+	},
+
+	// Support: Chrome <=50
+	// We should be able to just use this.element.prop( "isContentEditable" )
+	// but hidden elements always report false in Chrome.
+	// https://code.google.com/p/chromium/issues/detail?id=313082
+	_isContentEditable: function( element ) {
+		if ( !element.length ) {
+			return false;
+		}
+
+		var editable = element.prop( "contentEditable" );
+
+		return editable === "true" ? true :
+			editable === "false" ? false :
+			this._isContentEditable( element.parent() );
```

Comment:
What value does `editable` have in this case? I think this would be easier to understand (assuming its null, replace as needed):

```js
if ( editable === null ) {
  return this._isContentEditable( element.parent() );
}

return editable === "true";
```